### DISCUSSION TITLE
Why is the nightly version lower than the stable?

### DISCUSSION THREAD

**Discussion by StarkZarn** [netdata/netdata#16408] [Q&A] [answered]
I was checking to see if my nodes were updating and noticed that their version was lower than the latest release in Github. Manually running the netdata-updater.sh script, and the kickstart script yielded no additional updates. I then noticed that the version for the stable release is higher than nightly, which doesn't make sense to me. Am I missing features by being on the _default_ nightly branch? Nightly to me indicates more leading edge than "stable." What's up?

The Github badges corroborate what I'm seeing on my nodes vs the Github repo.
![image](https://github.com/netdata/netdata/assets/67663172/b97bff1d-6686-4baa-a0fc-7119fdd8ec33)

**Comment by Ferroin** [comment, marked as answer]
When we create a minor release, that involves creating a release branch. Any patch releases for a given minor release are made from that release branch and contain only _exactly_ those fixes that should be in the patch release, so there are no associated changes on the master branch resulting from a patch release. This is done both to minimize the risk of introducing new bugs, and to reduce any disruption for people working on the Netdata code that might occur when a patch release happens.

The nightly build versions though are literally just the latest tag on the master branch plus the number of commits since that tag (roughly equivalent to what you would get by running  `git describe` with the commit the nightly build was created from checked out). Because patch releases don’t have an associated tag on the master branch, nightlies never ‘see’ the patch releases.

**Reply by StarkZarn** [reply]
Got it. Thank you!
